BitwardenShared/Core/Vault/Repositories/VaultRepository.swift

```diff
@@ -926,13 +962,18 @@ class DefaultVaultRepository { // swiftlint:disable:this type_body_length
         let typesSecureNoteCount = activeCiphers.lazy.filter { $0.type == .secureNote }.count
         let typesSSHKeyCount = activeCiphers.lazy.filter { $0.type == .sshKey }.count
 
-        let types = [
-            VaultListItem(id: "Types.Logins", itemType: .group(.login, typesLoginCount)),
-            VaultListItem(id: "Types.Cards", itemType: .group(.card, typesCardCount)),
+        var types = [VaultListItem(id: "Types.Logins", itemType: .group(.login, typesLoginCount))]
+
+        // Only show the card section if there are cards and restrictItemTypes policy is not enabled.
```

Comment:
⛏️ I think you meant "or"
```suggestion
        // Only show the card section if there are cards or restrictItemTypes policy is not enabled.
```

Reply:
TY changed the logic and forgot the comment.